packages/react-lowcode/src/codegen/list.ts

```diff
@@ -20,12 +21,38 @@ export async function addColumn(typesSourceCode: string,
     options: InsertOptions): Promise<string | undefined>{
         
     const property: Property = getEntityProperty(typesSourceCode, options.property, options.entityName)[0]
+    const code = await io.readFile(sourceCode.fileName)
     let generatedSource = undefined
+    let isGrommet = false
+    if (code)
+        isGrommet = isGrommetRepository(code)
 
     if(property){
-        generatedSource = await insertColumn(sourceCode, 
-        {entityField: property, index: options.index}, 
-        io)
+        if (!isGrommet) {
+            generatedSource = await insertColumn(
+                sourceCode, 
+                {
+                    entityField: property, index: options.index
+                }, 
+                io
+            )
+        } else {
+            const myClassFile = parseGraphqlTypes(typesSourceCode)
+            const entity = sourceFileEntity(myClassFile, options.entityName)
+    
+            let ent : any = {};
+            ent.properties = [property]
+            ent.getName = entity?.getName
+            generatedSource = await insertColumnToDataTableGrommet(
+                sourceCode,
+                {
+                    entityField: property,
+                    index: options.index,
+                    entity: ent
```

Comment:
let's try to use `InsertOptions.entityName` without using `parseGraphqlTypes` and `sourceFileEntity`
we proably need to propagate this also to mui `insertColumn()`